config/dev.webpack.config.js

```diff
@@ -19,7 +20,27 @@ plugins.push(
     exposes: {
       './RootApp': resolve(__dirname, '../src/DevEntry'),
     },
-  })
+  }),
+  new ExtensionsPlugin(
+    {
+      extensions: [
```

Comment:
We don't want to have route for now? This will render directly in the content, which I guess is fine for now.

Reply:
I wasn't sure what we need here.

Reply:
If you want to have just a route in the navigation you can add, it should just show Marketplace in the navigation
```
{
      "type": "console.navigation/href",
      "properties": {
        "id": "dev-dynamic-route-1",
        "name": "Marketplace",
        "href": "/marketplace"
      }
}
```

Reply:
I thought we were talking about just having a direct access via url. 🤔 

But until the gateway is ready, we can let it as it is. There will be more changes otherwise, this would be fine for Demo purposes.

Reply:
Yeah, good point!